docs/src/main/paradox/testing.md

```diff
@@ -67,3 +67,19 @@ Scala
 
 Java
 :  @@snip [TestKitDocExample.java](/examples/src/test/java/jdocs/testkit/TestKitDocExample.java) { #testkit-assertion-import #testkit-sink-probe }
+
+## Testing with mocked Projection and SourceProvider
+
+To test a handler in isolation you may want to mock out the implementation of a Projection or SourceProvider so that you don't have to setup and teardown the associated technology as part of your _integration_ test.
```

Comment:
Hmm, yes I think I phrased this poorly. By integration test I mean a test that uses a real backend.  I'm trying to say the reason to use `TestProjection` and `TestSourceProvider` is to write a unit test (or at least a test that doesn't involve external technologies to be setup and torn down).

Do you think `s/integration/unit/` is good enough or should I rephrase this entirely?

> To test a handler in isolation you may want to mock out the implementation of a Projection or SourceProvider so that you don't have to setup and teardown the associated technology as part of your _unit_ test.